libs/orders/src/lib/components/order-list/order-list.spec.tsx

```diff
@@ -23,6 +23,17 @@ jest.mock('@vegaprotocol/react-helpers', () => ({
   }),
 }));
 
+jest.mock('@vegaprotocol/markets', () => ({
```

Comment:
I feel like this is going to become painful if we need to mock this for every test that will consume the markets map. Perhaps we can use a __mocks__ folder to mock this everywhere at once. Or maybe using context will be a cleaner solution